portafly/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:10-alpine as builder
```

Comment:
It would be nice to use CentOS to be the closest possible to productized build
But it is OK for now